### ISSUE TITLE
Feature Request - Session Threshold per CPU, minimum Peak hours

### ISSUE THREAD

**Issue by nigelwebster** [Azure/RDS-Templates#508]
we find in our experience that this is very limited.

with an 8 CPU VM we currently have 14 users allowed. this is just too much when it comes to scaling as by the time the next scale is run we could have up to 17 people during a logon surge in the morning. 

We cannot put in a decimal figure IE : 1.3 but I was wondering if there was an option to add a calculation in the script to then take a figure from this session threshold and then use decimals?

Also we find that in the morning when people have a logon surge we can overload the number of available servers. it would be great if we could have an addition to have a minimum number of servers available During Peak hours as this will start powering on some systems we know we are going to use at the start of the day. This will help load balance during a morning logon surge. as at present we are overloading the servers

**Comment by RoopChevuri** [comment]
Thanks for the feedback. We will review this for future release of the tool.

**Comment by AusInIndy** [comment]
If anyone was watching this thread I think I figured out a workaround.
 

I added this and it took "meaning if it is not valid it wont take", within the logic app for the autoscaling
 
"PercentageRuletoIncreaseCapacity": 50,

Then I set: "SessionThresholdPerCPU": 1," , so with my 12cpu machine it should allow for 12 users, but I dont want that I only want 6

With this when the first host is at 50% capacity, "No matter what the SessionThresholdPerCPU hosts is set to", it will start to fire up another host, then once that host is online it will load balance between the two. Then when both those hosts are at 50% it will fire up another host and load balance across all three hosts, in theory. We will see
